cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/services/RunRecordMonitorService.java

```diff
@@ -188,33 +200,38 @@ private void cleanupQueue() {
    * replication (i.e., {@link ProgramType#WORKER}) jobs.
    */
   private int getProgramsRunningCount() {
-    List<ProgramRuntimeService.RuntimeInfo> list = runtimeService
-        .listAll(ProgramType.WORKFLOW, ProgramType.WORKER, ProgramType.SPARK,
-            ProgramType.MAPREDUCE);
+    List<ProgramRuntimeService.RuntimeInfo> list =
+        runtimeService.listAll(
+            ProgramType.WORKFLOW, ProgramType.WORKER, ProgramType.SPARK, ProgramType.MAPREDUCE);
 
     int launchingCount = launchingQueue.size();
 
-    // We use program controllers (instead of querying metadata store) to count the total number of programs in
+    // We use program controllers (instead of querying metadata store) to count the total number of
+    // programs in
     // running state.
     // A program controller is created when a launch request is in the middle of starting state.
     // Therefore, the returning running count is NOT precise.
-    int impreciseRunningCount = (int) list.stream()
-        .filter(r -> isRunning(r.getController().getState().getRunStatus()))
-        .count();
+    int impreciseRunningCount =
+        (int)
+            list.stream()
```

Comment:
Do we really need cast and call on separate lines? Looks weird

Reply:
This is being caused by checkstyle autoformatter.

Reply:
Done